custom_components/xmrpool_stat/sensor.py

```diff
@@ -233,7 +233,9 @@ def unit_of_measurement(self) -> str:
     async def async_update(self):
         """Synchronize state with controller."""
         _LOGGER.debug("XmrPoolStatisticsSensorHashrate.async_update")
-        self._value = self._controller.GetValue(None, "hashrate").split()
+        value = self._controller.GetValue(None, "hashrate")
+        if value is not None:
```

Comment:
This problem happens when there are no miners if I understand issue correctly. Your change will not log error, but it'll not change last hashrate as well. 
I think you should add else with setting self._value to ['0', 'H']